lib/Page.js

```diff
@@ -165,21 +165,52 @@ Page.prototype.prepareTemplateData = function () {
 };
 
 /**
- * Records h1,h2,h3 headings into this.headings
- * @param renderedPage a page with its headings rendered
+ * Records headings in rendered page into this.headings
```

Comment:
To be consistent with the below method's comment:

` * Records headings inside rendered page into this.headings`